config/semileptonic_triggerSF/plot_options.py

```diff
@@ -1,343 +1,343 @@
 efficiency = {
     '2016_PreVFP' : {
         'Ele32_EleHT_pass': {
-            'hist_electron_pt' : {'ylim' : (0, 1)},
```

Comment:
Why is this dictionary necessary? It would be best to read the histogram config from the configuration. This is only replication, no?

Reply:
The reason for this dictionary to exist is to be able to configure the y-axis limits in data and MC trigger efficiency plots for each observable separately. These parameters are not directly related to the histogram itself but on the trigger efficiency as a function of the variable. At the moment `ylim` is set to a dummy value of `(0, 1)` but in the future it can be tuned to improve each plot.